pkg/project/project.go

```diff
@@ -155,6 +155,12 @@ func (p *projectBuilder) processConfigSection(properties map[string]map[string]s
 
 		location = p.standardizeLocation(location, folderPath)
 
+		//Introduce deprecation warning message when using the config type "application"
+		util.Log.Debug("Config type: " + getConfigType(folderPath))
```

Comment:
It would be better to check the `api`, which is returned from `getExtendedInformationFromLocation`. Then you don't need to parse the API from the folder path.